references/proxy-template/template-v1/package.json

```diff
@@ -7,12 +7,13 @@
     "test": "test"
   },
   "scripts": {
+    "deployTargetServer": "npm i && npx apigeetool createTargetServer --targetServerName $TARGET_SERVER_NAME -u $APIGEE_USER -p $APIGEE_PASS -o $APIGEE_ORG -e $APIGEE_ENV --targetHost $TARGET_HOST --targetPort $TARGET_PORT $APIGEETOOL_DEPLOY_TARGETSERVER_SSLOPTION -V",
     "deploy": "npm i && npx apigeetool deployproxy -u $APIGEE_USER -p $APIGEE_PASS -o $APIGEE_ORG -e $APIGEE_ENV -n $npm_package_name -V",
```

Comment:
we should maybe rename this to `deployProxy` now to be precise. WDYT?

Reply:
Yes agree with you.
So "deploy" becomes "deployProxy", right ?

Reply:
I have replaced **deploy** with **deployProxy**